Scramble libbtcore soname to the private one

--- a/libbtcore/CMakeLists.txt
+++ b/libbtcore/CMakeLists.txt
@@ -189,7 +189,7 @@
 if(WIN32)
 target_link_libraries(btcore ws2_32)
 endif(WIN32)
-set_target_properties(btcore PROPERTIES VERSION 8.0.0 SOVERSION 8 )
+set_target_properties(btcore PROPERTIES VERSION 8p.0.0 SOVERSION 8p )
 install(TARGETS btcore  ${INSTALL_TARGETS_DEFAULT_ARGS} )
 
 set(net_HDR  
