Index: ktorrent-4.3.1/ktorrent/ktorrent.desktop
===================================================================
--- ktorrent-4.3.1.orig/ktorrent/ktorrent.desktop	2013-01-14 17:12:24.000000000 +0000
+++ ktorrent-4.3.1/ktorrent/ktorrent.desktop	2014-03-04 19:02:40.209018100 +0000
@@ -118,7 +118,7 @@
 Icon=ktorrent
 Type=Application
 X-DocPath=ktorrent/index.html
-MimeType=application/x-bittorrent;application/x-torrent;
+MimeType=application/x-bittorrent;application/x-torrent;x-scheme-handler/magnet;
 X-DBUS-StartupType=Unique
 X-DBUS-ServiceName=org.ktorrent.ktorrent
 Comment=A BitTorrent program for KDE
