From: Modestas Vainius <modestas@vainius.eu>
Subject: [PATCH] general/fix_target_link_libraries

Do not link with excess libraries.

Signed-off-by: Modestas Vainius <modestas@vainius.eu>

--- a/plugins/scripting/CMakeLists.txt
+++ b/plugins/scripting/CMakeLists.txt
@@ -20,7 +20,6 @@ target_link_libraries(
     ${KDE4_KDECORE_LIBS}
     ${KDE4_KDEUI_LIBS}
     ${KDE4_KROSSCORE_LIBS}
-    ${KDE4_KROSSUI_LIBS}
     ${KDE4_KPARTS_LIBS}
     ${QT_QTCORE_LIBRARY}
     )
